### ISSUE TITLE
Bug: Devtools in Firefox does not allow adding props or editing props/state/hooks

### ISSUE THREAD

**Issue by hbarcelos** [facebook/react#18544]
Version:
- React @ `16.13.1`
- Firefox @ `75.0` (version for Manjaro Linux)
- DevTools version `4.6.0-6cceaeb67`

**IMPORTANT:** I have the same React DevTools version `4.6.0-6cceaeb67` installed in Google Chrome Beta 81 and it's working as expected.

## Steps To Reproduce

1. Open React DevTools Components tab
2. Select a component with props/state
3. Try to add props or edit props, state or hooks.

Here's a screen grab of the issue:

![react_devtools_issue](https://user-images.githubusercontent.com/16565602/78829763-516b7c80-79bd-11ea-9e24-62b6e16a8ecf.gif)
Between the clicks, I'm typing on my keyboard, but nothing happens.

## The current behavior

All keys typed in this tab are ignored.

## The expected behavior

I should be able to add props or alter props, state, hooks, etc.

**Comment by daniel-sudz** [comment]
Cannot reproduce on windows: 
 ` ` `
"react": "^16.13.1"
 "react-dom": "^16.13.1"
 ` ` `

 ` ` `
Firefox 74.0
 ` ` `

 ` ` `
Dev-tools 4.6.0
 ` ` `

If this issue specific to manjaro/linux?

**Comment by AlexKorobko** [comment]
Distributor ID: Ubuntu
Description: Ubuntu 18.04.4 LTS
Release:	18.04
Codename: bionic

"react": "^16.13.1"
"react-dom": "^16.13.1"

Firefox 75.0

Changing props/state works.

![Dev-tools-issue](https://user-images.githubusercontent.com/47081998/79004257-11db9680-7b5d-11ea-9109-fa9a80704298.gif)

**Comment by hbarcelos** [comment]
So, I've done the following:

- Checked for patch updates for Firefox Manjaro (none by the time I'm writing).
- Rebooted my notebook
- Uninstalled and reinstalled React Developer Tools addon for Firefox (without trying to remove any related config though).

Now I get this for my app:

This is Firefox: the component tree won't even load.
![image](https://user-images.githubusercontent.com/16565602/79027804-08a1f800-7b64-11ea-9502-8cb6c52aa1c4.png)
**This happens only in my app, others I testes seem to be working fine**

This is Firefox in Private Mode:
![image](https://user-images.githubusercontent.com/16565602/79027831-23746c80-7b64-11ea-90ca-91b7e03b48eb.png)
**This happens only in every app I tried**

This is Chrome (which still works just fine):
![image](https://user-images.githubusercontent.com/16565602/79027922-764e2400-7b64-11ea-87ef-7695b2976c64.png)

---

### Further Information

- My app uses the latest `parcel@1.x.x` as bundler.
- If I remove everything from the app, leaving just an empty div, the problem still persists.

This was a long week for me, I'll keep digging and let you know if I find anything, but I probably won't have any news before next Monday.

**Comment by AlexKorobko** [comment]
Created one component react app with : "parcel-bundler": "^1.12.4".
Works fine in mozilla and chrome.
![image](https://user-images.githubusercontent.com/47081998/79050635-13c45900-7c34-11ea-8499-cf5a124da8ea.png)

**Comment by daniel-sudz** [comment]
I think we should close this issue because its not possible to reproduce on a standard setup.

**Comment by hbarcelos** [comment]
Quick update:

I completely removed Firefox from my PC, along with all cache and metadata and installed it again from scratch.

I chose the Firefox Developer Version this time:
![Firefox Version](https://user-images.githubusercontent.com/16565602/79363780-baec0d80-7f1e-11ea-9a30-8e869543b56c.png)

The behavior I get now is:
- I still cannot edit content properly
- Some keys like <kbd>Backspace</kbd>, <kbd>Space</kbd>, <kbd>"</kbd>, <kbd>`</kbd> work properly
- Alphanumeric keys don't work at all
- However, if I copy the value from some place else, pasting in the field works fine

**Comment by bvaughn** [comment]
@hbarcelos Do you, by chance, have the "search as you type" feature enabled?

**Comment by hbarcelos** [comment]
Bulls eye, @bvaughn!

I did have the "Search for text when you start typing" feature on (I don't remember setting this on though).

After I disabled it, the dev tools started working again.

The weird part is that I haven't changed my configs in a while (I'd say for more than 2 years) and the day I posted here was the first time I ever experienced this issue. Maybe something in that regard changed in the recent Firefox releases.

> With regard to this, have you tried reloading the page? Occasionally the Components tab doesn't populate (not sure of the reason) but a reload always fixes it.

Regarding that, yes, for sure... Reloading and fresh reloading were the first things I tried. This issue went away with the clean install though.

Thank you very much!
